src/verify/index.ts

```diff
@@ -2,6 +2,10 @@ import { timingSafeEqual } from "crypto";
 import { Buffer } from "buffer";
 import { sign } from "../sign/index";
 
+const getAlgorithm = (signature: string) => {
+  return signature.indexOf("sha256=") === 0 ? "sha256" : "sha1";
+};
```

Comment:
@gr2m is it fine the check at this stage the algorithm to be applied or you want to be decided before and change the signature of verify with an `algorithm?` parameter?

Reply:
Your implementation works good to me 👍🏼